firebase-appdistribution/src/main/java/com/google/firebase/appdistribution/TesterSignInManager.java

```diff
@@ -45,7 +45,7 @@
 class TesterSignInManager {
   private static final String TAG = "TesterSignInManager:";
   private static final String SIGNIN_REDIRECT_URL =
-      "https://appdistribution.firebase.google.com/pub/testerapps/%s/installations/%s/buildalerts?appName=%s&packageName=%s";
+      "https://appdistribution.firebase.google.com/pub/testerapps/%s/installations/%s/buildalerts?appName=%s&packageName=%s&newRedirectScheme=appdistribution";
```

Comment:
I think `newRedirectScheme=true` would make more sense to me, since we don't actually use the value, and the param name sounds like it should have a boolean assigned to it anyway.